Sources/SwiftDocC/Indexing/Navigator/NavigatorIndex.swift

```diff
@@ -707,6 +707,22 @@ extension NavigatorIndex {
                 multiCuratedUnvisited.remove(normalizedIdentifier)
             }
             
+            // Check if the render node has a variant for Objective-C
+            let objCVariantTrait = renderNode.variantOverrides?.values.flatMap({ $0.traits }).first { trait in
+                switch trait {
+                case .interfaceLanguage(let language):
+                    return InterfaceLanguage.from(string: language) == .objc
```

Comment:
Is the interface language for a pure C library `objc`?

Reply:
This will be decided in the implementation of the compiler that will emit symbol graph files, we'll want to extend the support for pure C libraries then.